src/pipecat/processors/frameworks/rtvi.py

```diff
@@ -377,6 +385,11 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
             await self.push_frame(frame, direction)
         elif isinstance(frame, TransportMessageFrame):
             await self._message_queue.put(frame)
+        elif isinstance(frame, RTVIActionFrame):
+            if not frame.message_id:
+                # The response isn't going anywhere anyway
+                frame.message_id = "webhook"
```

Comment:
I would remove this `if` statement. Then in `_handle_action()` we could accept `str | None` and only send the `RTVIActionResponse` if `request_id` is not `None`.

```
        result = await action.handler(self, action.service, arguments)
        if request_id:
            message = RTVIActionResponse(id=request_id, data=RTVIActionResponseData(result=result))
            await self._push_transport_message(message)
```